### PR TITLE
Generated eqs

### PR DESCRIPTION
**Pull request by Shreyas4991**
Fix the import stuff with automatically generated files.
Add a Generated.lean file under equational_theories. 
Fix the CONTRIBUTING.md documentation

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Proofs generated programmatically are also welcome. If you do this, you are enco
 - Then add a single Lean file (e.g., "equational_theories/Generated/MyTechnique.lean") that references all your theorems.
 - Add a README (e.g., "equational_theories/Generated/MyTechnique/README.md") to document how to reproduce your automatically generated theorems (to the extent possible, for randomized algorithms).
 - In [.gitattributes](.gitattributes) add `equational_theories/Generated/MyTechnique.lean linguist-generated` for all automatically generated files.
+- In the file `equational_theories/Generated.lean` add the import line `import equational_theories.Generated.MyTechnique` replacing "MyTechnique with the name of your folder.
```
Comment (carlini) [inline CONTRIBUTING.md:71]: Typo: missing end double quote.

Comment (Shreyas4991) [reply CONTRIBUTING.md:71]:
```suggestion
- In the file `equational_theories/Generated.lean` add the import line `import equational_theories.Generated.MyTechnique` replacing "MyTechnique" with the name of your folder.
```

File: equational_theories/Generated.lean
Comment (vlad902) [inline equational_theories/Generated.lean:1]: Should we add `Constant.lean` and `Singleton.lean` while we're here?

Comment (Shreyas4991) [reply equational_theories/Generated.lean:1]: Agreed

**Pull request merged**